tests/perf/pubsub_publish_http/test.js

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import http from "k6/http";
+import { check } from "k6";
+import crypto from "k6/crypto";
+
+const KB = 1000
+const MAX_MS_ALLOWED = 1
+// padd with leading 0 if <16
+function i2hex(i) {
+    return ('0' + i.toString(16)).slice(-2);
+}
+
+function randomStringOfSize(size) {
+    const bytes = crypto.randomBytes(size);
+    const view = new Uint8Array(bytes);
+    return view.reduce(function (memo, i) { return memo + i2hex(i) }, '')
+}
+
+const scenarioBase = {
+    executor: "constant-arrival-rate",
+    rate: 1,
+    preAllocatedVUs: 2,
+    maxVUs: 50,
+}
+const delaysMs = [1, 5, 50, 100, 1000]
+const messageSizeKb = [2, 31]
+const brokers = __ENV.BROKERS.split(",")
+const samples = [200]
+
+let scenarios = {}
+let thresholds = {
+    checks: ['rate==1'],
+}
+
+for (const messageSizeKbIdx in messageSizeKb) {
+    const msgSize = messageSizeKb[messageSizeKbIdx]
+    const msgStr = randomStringOfSize(msgSize * KB)
```

Comment:
This is technically doubling the length however, as it's hex-encoding the message of msgSize. If I'm not mistaken.

Reply:
you are right, I'm afraid that using the UTF-8 byte value won't work because of encoding. So, how about divide the input by 2?

Reply:
Sure that works